packages/charts/src/chart_types/goal_chart/state/selectors/tooltip.ts

```diff
@@ -47,7 +47,7 @@ export const getTooltipInfoSelector = createCustomCachedSelector(
           key: spec.id,
         },
         value,
-        formattedValue: `${value}`,
+        formattedValue: valueFormatter ? valueFormatter(value) : `${value}`,
```

Comment:
From your typings `valueFormatter` is always available, no need to check for its existence.
```suggestion
        formattedValue: spec.valueFormatter(value),
```

Reply:
ok that makes sense - d41cb16 for changes thanks